modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,535 @@
+
+For information on the **Real-Time Collaboration** plugin, see: xref:rtc-introduction.adoc[Real-Time Collaboration].
+
+=== Spell Checker Pro 3.1.0
+
+The {productname} 6.2 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
+
+Spell Checker Pro 3.1.0 adds one change and fixes two bugs:
+
+==== The `SpellcheckerLanguageChanged` event is now dispatched at plugin initialization
+
+Spell Checker Pro now fires the `SpellcheckerLanguageChanged` event when it is initialized, making the language Spell Checker Pro has been configured to use available for querying.
+
+This enables other plugins to check for the currently configured language.
+
+==== In some specific cases, copying and pasting in tables did not work as expected due to Spell Checker Pro incorrectly moving the selection
+
+In some circumstances, Spell Checker Pro would silently, and incorrectly, extend a selection. When the extended selection was then copied-and-pasted, unexpected results occurred.
+
+For example, in some circumstances the selection appeared to be the contents of a single table cell, but Spell Checker Pro incorrectly extended the selection to include an HTML markup element outside the table. When this wa pasted, the incorrectly added element made the paste behave incorrectly.
+
+With this update, the way elements are traversed when selections are being made has been changed. Consequently, incorrect selection extension no longer occurs, and copying-and-pasting content from tables works as expected.
+
+==== Spellchecking incorrectly triggered on all `change` events
+
+Spell Checker Pro should trigger on any `paste` event, whether the event comes from a plugin (such as Paste or PowerPaste) or a native browser action. Previously, however, Spell Checker Pro was listening to the wrong event — the `change` event — to cover this case.
+
+As a consequence, Spell Checker Pro triggered on all `change` events.
+
+With this update, Spell Checker Pro now listens for input events and only triggers when content is inserted into a {productname} document.
+
+For information on the **Spell Checker Pro** plugin, see: xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro].
+
+[[accompanying-premium-skins-and-icon-packs-changes]]
+== Accompanying Premium Skins and Icon Packs changes
+
+The {productname} 6.2 release includes an accompanying release of the **Premium Skins and Icon Packs**.
+
+=== Premium Skins and Icon Packs
+
+The **Premium Skins and Icon Packs** release includes the following update:
+
+The **Premium Skins and Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} 6.2 skin, Oxide.
+
+For information on using premium skins and icon packs, see: xref:premium-skins-and-icons.adoc[Premium Skins and Icon Packs].
+
+
+[[additions]]
+== Additions
+
+{productname} 6.2 also includes the following additions:
+
+=== A new `text_patterns_lookup` option to provide additional text patterns dynamically.
```

Comment:
```suggestion
=== A new `text_patterns_lookup` option to provide additional text patterns dynamically
```
No full stops on headings normally. They are inconsistent within this file at the very least.